Added part `Docker` to docs (#12403)
ydb-platform/ydb: ydb/docs/en/core/reference/docker/index.md

# {{ ydb-short-name }} Docker container reference

This section provides detailed information about working with {{ ydb-short-name }} when delivered as a Docker container.

The {{ ydb-short-name }} Docker container reference documentation includes the following sections:

* [{#T}](naming.md)
* [{#T}](install.md)
* [{#T}](start.md)

> * [{#T}](environment.md)

Author: ```suggestion
* [{#T}](configuration.md)
```

> # Help for working with {{ydb-short-name }} in Docker
> 
> This section contains more information about working with {{ ydb-short-name }}, which is delivered as a Docker container.
> 
> The Docker container documentation contains the following sections:

Reviewer: ```suggestion
# {{ ydb-short-name }} Docker container reference

This section provides detailed information about working with {{ ydb-short-name }} when delivered as a Docker container.

The {{ ydb-short-name }} Docker container reference documentation includes the following sections:
```

> * [{#T}](install.md)
> * [{#T}](start.md)
> * [{#T}](environment.md)

Reviewer: ```suggestion
* [{#T}](naming.md)
* [{#T}](install.md)
* [{#T}](start.md)
* [{#T}](environment.md)
* [{#T}](cleanup.md)
```
